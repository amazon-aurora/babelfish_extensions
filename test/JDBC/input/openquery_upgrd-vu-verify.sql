SELECT * FROM OPENQUERY(bbf_server, 'SELECT 123')
GO

-- check whether query timeout value is getting persisted after the upgrade
SELECT name, query_timeout FROM sys.servers WHERE name = 'bbf_server_1'
GO

-- should throw error as it takes more than 5 seconds to run
SELECT * FROM OPENQUERY(bbf_server_1, 'select CAST(pg_sleep(5) AS text)')
GO

-- SELECT * FROM openquery_vu_prepare__openquery_view
GO

EXEC openquery_upgrd_vu_prepare__openquery_proc
GO

SELECT openquery_upgrd_vu_prepare__openquery_func()
GO

-- Call OPENQUERY from a database other than master
USE tempdb
SELECT * FROM OPENQUERY(bbf_server, 'SELECT ''Called from tempdb''')
GO

CREATE DATABASE openquery_db
USE openquery_db
GO

SELECT * FROM OPENQUERY(bbf_server, 'SELECT ''Called from openquery_db''')
GO

USE master
DROP DATABASE openquery_db
GO
