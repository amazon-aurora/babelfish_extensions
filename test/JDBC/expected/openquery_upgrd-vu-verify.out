SELECT * FROM OPENQUERY(bbf_server, 'SELECT 123')
GO
~~START~~
int
123
~~END~~


-- check whether query timeout value is getting persisted after the upgrade
SELECT name, query_timeout FROM sys.servers WHERE name = 'bbf_server_1'
GO
~~START~~
varchar#!#int
bbf_server_1#!#1
~~END~~


-- should throw error as it takes more than 5 seconds to run
SELECT * FROM OPENQUERY(bbf_server_1, 'select CAST(pg_sleep(5) AS text)')
GO
~~START~~
text
~~ERROR (Code: 33557097)~~

~~ERROR (Message: TDS client library error: DB #: 20003, DB Msg: server connection timed out, OS #: 0, OS Msg: Success, Level: 6)~~


-- SELECT * FROM openquery_vu_prepare__openquery_view
GO

EXEC openquery_upgrd_vu_prepare__openquery_proc
GO
~~START~~
int
2
~~END~~


SELECT openquery_upgrd_vu_prepare__openquery_func()
GO
~~START~~
int
1
~~END~~


-- Call OPENQUERY from a database other than master
USE tempdb
SELECT * FROM OPENQUERY(bbf_server, 'SELECT ''Called from tempdb''')
GO
~~START~~
varchar
Called from tempdb
~~END~~


CREATE DATABASE openquery_db
USE openquery_db
GO

SELECT * FROM OPENQUERY(bbf_server, 'SELECT ''Called from openquery_db''')
GO
~~START~~
varchar
Called from openquery_db
~~END~~


USE master
DROP DATABASE openquery_db
GO

